### PR TITLE
(#125) Fix overall code coverage

### PR DESCRIPTION
**Pull request by seyilonge-nci**
(#125) Fix overall code coverage](https://github.com/NCIOCPL/sitewide-search-app/pull/159/commits/451534b44220ab48d2ed0671fa62999e00109fd6) 

* Added pre-merge script to setup combined coverage directories and copy
  coverage files from cypress and jest.
* Added merged coverage specific config, as the nyc section in
  package.json is used by cypress.
* Added text reporting after ALL integration tests finish.
  When text reporting is enabled Cypress reports after each feature.
* Added reporting for full merged coverage at the end with
  coverage checking set to 80%.
* Boosted coverage to 82%
* Removed polyfill for ie11
* Updated config for istanbul plugin (removed from `babel.config.js`, added to `webpack.config.js` for non-PROD env)

Closes https://github.com/NCIOCPL/sitewide-search-app/issues/125

(#125) Linter Fixes for code coverage update](https://github.com/NCIOCPL/sitewide-search-app/pull/159/commits/cc61fae5ac3aa3e37f7e6914f07a20a089bba2e7) 

* Ran `npm run format` to fix linting issues
* Removed unused variables noted in linter log

### CODE REVIEW

**Review by sarinapadilla — APPROVED** [review]
`react-app-dev` is working, so this passes for me!

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
+
 import 'react-app-polyfill/ie11';
```
Comment (arcepaul) [inline src/index.js:3]: this may not be necessary anymore?

Comment (seyilonge-nci) [reply src/index.js:3]: Quite true, thank God! Would just our testing in place along with manual testing suffice for the removal?

Comment (seyilonge-nci) [reply src/index.js]: @arcepaul Removed.

**Review by adriancofie — APPROVED** [review]

**Review by arcepaul — APPROVED** [review]

**Pull request merged**